client/src/app/site/pages/meetings/pages/motions/motions.subscription.ts

```diff
@@ -67,7 +67,13 @@ export const getMotionListSubscriptionConfig: SubscriptionConfigGenerator = (id:
                         follow: [
                             {
                                 idField: `meeting_user_id`,
-                                follow: [{ idField: `user_id`, ...UserFieldsets.FullNameSubscription }]
+                                follow: [
+                                    {
+                                        idField: `user_id`,
+                                        fieldset: [...UserFieldsets.FullNameSubscription.fieldset, `meeting_user_ids`]
```

Comment:
Unnecessary subscription to meeting user ids should be prevented. Removing this can be tracked in #2789.

Reply:
Not a change request just a comment.